packages/wonder-blocks-button/src/components/button-core.js

```diff
@@ -162,7 +162,7 @@ export default class ButtonCore extends React.Component<Props> {
                 <StyledButton
                     type={type || "button"}
                     {...commonProps}
-                    disabled={disabled}
+                    aria-disabled={disabled}
```

Comment:
How does `aria-disabled` differ from `disabled`?  Should we be using `aria-disabled` in other clickable components like `IconButton` and the various drop-downs and the items the contain?

Reply:
Yeah that's a good question. I remember listing out some of the elements that would be nice to update with this pattern, but probably a good follow up ticket after this is landed.